docs/ref/databases.txt

```diff
@@ -171,6 +171,19 @@ If you need to add a PostgreSQL extension (like ``hstore``, ``postgis``, etc.)
 using a migration, use the
 :class:`~django.contrib.postgres.operations.CreateExtension` operation.
 
+Server side cursors
+-------------------
+
+When using :ref:`queryset_iterator`, Django opens a :ref:`server-side cursor
+<psycopg2:server-side-cursors>`. By default, when PostgreSQL plans a cursor
+query, it assumes that only the first 10% of the results will be fetched. This
+results in a fast-start plan, but can take excessive time if the whole results
```

Comment:
I'm not sure what "fast-start" means. Is it a technical term? A Google search was unhelpful. 
"if the whole results" -- what about if more than 10% of results are retrieved?